winrm/winrm_test.go

```diff
@@ -0,0 +1,130 @@
+// copyright 2016 canonical ltd.
+// copyright 2016 cloudbase solutions
+// licensed under the lgplv3, see licence file for details.
+
+package winrm_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/juju/utils/winrm"
+
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+)
+
+func TestAll(t *testing.T) {
+	gc.TestingT(t)
+}
+
+type WinRMSuite struct{}
+
+var _ = gc.Suite(&WinRMSuite{})
+
```

Comment:
I think these tests all need to be updated now. There's no ResetClientCert, LoadClientCert, ClientCert, ClientKey, etc. Probably you should just have pre-generated certs/keys checked in the winrm_test package, and use them in your tests. Avoid generating new certs/keys for each test (or even the suite, if possible), because it's expensive and slows down tests. Not avoidable in all cases though.